packages/styles/src/menu-dropdown/menu-dropdown.scss

```diff
@@ -0,0 +1,95 @@
+@import "../core/_dev";
+
+.as-menu-dropdown {
+  --as-dropdown-menu-hover-color: #D3F6E6;
+  position: relative;
+
+  &--input {
+    width: 100%;
+    min-height: 32px;
+    margin: 0;
+    padding: get-spacing(2);
+    border: 0;
+    background: var(--as-color-ui-01, #{$color-ui-01});
+    color: var(--as-color-primary, #{$color-primary});
+    font: $font-body;
+    text-align: left;
+    cursor: pointer;
+
+    &:hover,
+    &:focus {
+      // Pending focus
+    }
+  }
+
+  &--options {
+    position: absolute;
+    z-index: 3;
+    width: 100%;
+    height: 0;
+    margin: 0 auto;
+    padding: 0;
+    overflow: hidden;
+    border-radius: 4px;
+    box-shadow: $shadow-8;
+
+    li {
+      display: flex;
+      align-items: center;
+      width: 100%;
+      transition: background-color 0.3s;
+      border: 0;
+      border-bottom: 1px solid var(--as-color-ui-04, #{$color-ui-04});
+      outline: none;
+      background: var(--as-color-ui-01, #{$color-ui-01});
+      color: var(--as-color-type-01, #{$color-type-01});
+      font: $font-body;
+      list-style: none;
+      cursor: pointer;
+
+      &:first-child {
+        border-top: 1px solid #F0F0F0;
+        border-radius: 4px 4px 0 0;
+      }
+
+      &:last-child {
+        border-radius: 0 0 4px 4px;
+      }
+
+      &:hover,
+      &:focus-within {
+        background: var(--as-dropdown-menu-hover-color, #D3F6E6);
```

Comment:
SASS variable for fallback?

Reply:
[same](https://github.com/CartoDB/airship/pull/152#discussion_r214301170)

Reply:
You've used it twice already xD
I think we can put it as a SASS variable on the top of the file with that colour.
It is `#47DB99` (which is an Airship color) with an alpha channel of 16, but as far as I know there's not a syntax for `rgba(#47DB99, 16)`. I don't know if we can interpolate a string like `#47DB99FF` with `#{$color-complementary}FF` to a variable and use it as background.